ce/marketing/developer/event-management-web-application.md

```diff
@@ -18,88 +18,114 @@ manager: kvivek
 ---
 
 # Build and host a custom event portal
+The Dynamics 365 Event Management feature helps you to organize, manage and process the events. When you install the Event Management solution, you will get an **Event Portal** which gives the users a web application where they can get to know more about the event details. More information: [Event Portal](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/marketing/set-up-event-portal).
 
-The Dynamics 365 Event Management feature helps you to organize, manage and process the events. When you install the Event Management solution, you will get an **Event Portal** which gives the users a web application where they can know more about the event details. More information: [Event Portal](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/marketing/set-up-event-portal).
-
-
-The Event Management web application consists of two parts:
+The Event Management event portal consists of two parts:
 - Angular frontend
```

Comment:
Not necessarily Angular, it's just an example can be pure vanilla js

Reply:
Thanks for the hint, changes it!